package-lock.json

```diff
@@ -1,13722 +1,8 @@
 {
   "name": "planet-web2",
   "version": "0.2.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Please revert the change of this file. We are using lockfileVersion 2 now. Please upgrade your npm version to at least 7.x, better 8.x.